Gemfile

```diff
@@ -31,3 +31,5 @@ group :test do
 end
 
 gemspec
+
+gem 'ruby2_keywords', '~> 0.0.1'
```

Comment:
This perhaps ought to live in the gemspec itself?

Reply:
🤦‍♂️ fixed